corehq/apps/api/resources/v0_5.py

```diff
@@ -314,12 +333,20 @@ def obj_create(self, bundle, **kwargs):
     def obj_update(self, bundle, **kwargs):
         bundle.obj = CommCareUser.get(kwargs['pk'])
         assert bundle.obj.domain == kwargs['domain']
+        send_confirmation_email = True if bundle.data.pop('send_confirmation_email_now', None) == "True" else False
         user_change_logger = self._get_user_change_logger(bundle)
         errors = self._update(bundle, user_change_logger)
         if errors:
             formatted_errors = ', '.join(errors)
             raise BadRequest(_('The request resulted in the following errors: {}').format(formatted_errors))
         assert bundle.obj.domain == kwargs['domain']
+
+        if (toggles.TWO_STAGE_USER_PROVISIONING.enabled(kwargs['domain'])
+                and not bundle.obj.is_account_confirmed and send_confirmation_email):
```

Comment:
we should also validate that if `send_confirmation_email_now` is True, then `is_account_confirmed` has to be`False`. Otherwise, respond with a 409 ~ “account is already confirmed”

Reply:
Am I missing something? `and not bundle.obj.is_account_confirmed and send_confirmation_email)` is doing that check I believe

Reply:
gotcha - refactored in [4548870](https://github.com/dimagi/commcare-hq/pull/36675/commits/45488700ab064aaafaed5e6e2c591ad347bf8814)